supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc

```diff
@@ -0,0 +1,245 @@
+
+*Use it*: yes
+
+*Incorrect forms*: DM, directory manager
+
+*See also*:
+
+[discrete]
+[[directory-server]]
+==== image:images/yes.png[yes] Directory Server (noun)
+*Description*: The short product name of Red Hat Directory Server. In the title of guides, use the full product name Red Hat Directory Server and, elsewhere, the short name Directory Server. Because it is the product name, both words start with a capital letter. Additionally, this distinguishes the Red Hat Directory Server product from other directory servers.
+
+*Use it*: yes
+
+*Incorrect forms*: directory server
+
+*See also*: xref:red-hat-directory-server[Red Hat Directory Server]
+
+[discrete]
+[[distinguished-name]]
+==== image:images/yes.png[yes] distinguished name (noun)
+*Description*: A distinguished name (DN) is a sequence of relative distinguished names (RDN) connected by commas. A DN defines the unique location of an entry in the LDAP directory. Use distinguished name on the first usage and then the abbreviation DN subsequently.
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*: 
+
+[discrete]
+[[hub]]
+==== image:images/yes.png[yes] hub (noun)
+*Description*: In an LDAP replication environment, hubs receive data from a supplier and replicate the data to consumers.
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*: 
+
+[discrete]
+[[ldap]]
+==== image:images/yes.png[yes] LDAP (noun)
+*Description*: The Lightweight Directory Access Protocol (LDAP) defines an industry standard for accessing and maintaining directory servers, such as Red Hat Directory Server. By default, the LDAP protocol is unencrypted. Do not expand.
+
+*Use it*: yes
+
+*Incorrect forms*: Lightweight Directory Access Protocol
+
+*See also*: xref:ldaps[LDAPS], xref:starttls[STARTTLS]
+
+[discrete]
+[[ldaps]]
+==== image:images/yes.png[yes] LDAPS (noun)
+*Description*: The Lightweight Directory Access Protocol over Secure Socket Layer (LDAPS) uses the TLS protocol to encrypt LDAP traffic. Do not expand.
+
+*Use it*: yes
+
+*Incorrect forms*: Lightweight Directory Access Protocol over Secure Socket Layer
+
+*See also*: xref:ldap[LDAP], xref:starttls[STARTTLS]
+
+[discrete]
+[[master]]
+==== image:images/no.png[no] master (noun)
+*Description*: Do not use master to refer to a supplier.
+
+*Use it*: no
+
+*Incorrect forms*: 
+
+*See also*: xref:supplier[supplier]
+
+[discrete]
+[[object-class]]
+==== image:images/yes.png[yes] object class (noun)
+*Description*: Object classes in an entry control which attributes are optional and which are required. Write as two words when you refer to object classes in general.
+
+*Use it*: yes
+
+*Incorrect forms*: objectClass
+
+*See also*: xref:objectclass[objectClass]
+
+[discrete]
+[[objectclass]]
+==== image:images/yes.png[yes] objectClass (noun)
+*Description*: The objectClass attribute in an LDAP entry stores the object classes of this entry. 
```

Comment:
Just curious, if you were writing about this objectClass, would you mark it up in monospace as `objectClass`? If so, I would do that here too. (And if not, then ignore this! :) )

Reply:
I'm not the RHDS writer, so not sure! I can ask Marc if you'd prefer me to follow up on this.

Reply:
Nah, not a big deal, thanks